### PR TITLE
Improvised Quivers and Holster

### PR DESCRIPTION
**Pull request by maibock-ell**
#### Summary
SUMMARY: Content "Two new improvised quivers for the early game, as well as a holster."
#### Purpose of change
I believe that the early game should have more options for storing firearms, and ammunition for bows/crossbows. Obtaining holsters and quivers with early game characters is dependant on RNG. Decided to make these as well due to it being easy for a first PR.
#### Describe the solution
Three new items are added as craftables whose recipes are known by default. Two are made from boots, a thigh-high quiver and a knee-high quiver respectively. A flip-flop holster is also here. All three require a leather or webbing belt, and are worse than their non-improvised relatives.
#### Describe alternatives you've considered
As an altenative, one could make birchbark quivers or gamble with RNG in cities. But for those who are in a city and not innawoods, one could make boot quivers. 

#### Testing
I have looked over the JSON at least four times for formatting errors, and so on. I also have done a test in game to make sure that the recipes are able to be used, and so on. The recipes for these items are known by default.

#### Additional Context
My first PR, so there are bound to be errors in the JSON.

### CODE REVIEW

**Comment by dissociativity** [comment]
it just occurred to me, you're only using one boot, so you should in theory get scrap material for the other boot left over as a crafting result, correct?

Seems like plenty of materials are disappearing from reality through the recipe.

**Comment by maibock-ell** [comment]
> it just occurred to me, you're only using one boot, so you should in theory get scrap material for the other boot left over as a crafting result, correct?
> 
> Seems like plenty of materials are disappearing from reality through the recipe.

Oh! You seemed to read what I had said wrong. To be fair, it might have been confusing. There are actually two kinds of boot-quivers in this PR, "knee-high boot quiver" and "thigh-high boot quiver".  The difference is that the knee-high boot quiver can only hold bolts due to its size.

**Comment by int-ua** [comment]
What's this `h.txt`?

**Comment by maibock-ell** [comment]
> What's this `h.txt` supposed to be?

Huh, have no clue how that managed to get into the PR.

File: data/json/items/armor/ammo_pouch.json [hunk 1]
```diff
@@ -507,5 +507,47 @@
...
+    "bashing": 2,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "looks_like": "boots_larmor",
+    "color": "black",
+    "covers": [ "LEG_EITHER" ],
+    "coverage": 10,
+    "encumbrance": 16,
+    "material_thickness": 1,
+    "pocket_data": [ { "ammo_restriction": { "arrow": 40, "bolt": 40 }, "moves": 25 } ],
```
Comment (Maleclypse) [inline data/json/items/armor/ammo_pouch.json:529]:
```suggestion
    "pocket_data": [ { "ammo_restriction": { "arrow": 40, "bolt": 40 }, "moves": 25 } ],
```
Sources on historic quivers suggest between 20 and 30 arrows in a made for arrows quiver.  I think we should assume significantly less for a makeshift quiver.

File: data/json/items/armor/holster.json [hunk 2]
```diff
@@ -311,5 +311,36 @@
     ],
     "use_action": { "type": "holster" },
     "flags": [ "WAIST", "OVERSIZE" ]
+  },
+  {
+    "id": "flip_flop_holster",
+    "type": "ARMOR",
+    "name": { "str": "Flipflop Holster" },
+    "description": "A improvised solution for the lack of a holster.  Rather comedic, but gets the job done albeit with more encumbrance when commpared to a regular holster.",
```
Comment (Maleclypse) [inline data/json/items/armor/holster.json:319]: Is there a source for this working?

File: h.txt [hunk 3]
```diff
@@ -0,0 +1,193 @@
+!src/chkjson
```
Comment (Maleclypse) [inline h.txt:1]: Delete this file?

**Review by Maleclypse — CHANGES_REQUESTED** [review]

**Review by kevingranade — CHANGES_REQUESTED** [review]
No problem in broad strokes, but the existing feedback needs to be addressed.

**Comment by maibock-ell** [comment]
Haven't had the time to properly work on this PR, so I'm closing it for now.

**Pull request closed without merging**